rskj-core/build.gradle

```diff
@@ -1,10 +1,156 @@
 plugins {
     id 'application'
+    id 'checkstyle'
+    id "com.diffplug.spotless" version "6.13.0"
 }
 
 apply plugin: 'maven-publish'
 apply plugin: 'jacoco'
 
+import java.nio.file.Paths
+
+// Define static timestamp
+def staticTimestamp = '2024-10-17T10:24:00'
+ext.timestamp = staticTimestamp
+
+def filteredFiles = getFilteredFiles()
+
+checkstyle {
+    toolVersion = '8.45'
+    configFile = file("$rootDir/config/checkstyle/checkstyle.xml")
+}
+
+spotless {
+    java {
+        googleJavaFormat()
+        indentWithTabs(2)
+        indentWithSpaces(4)
```

Comment:
Maybe we could also add `trimTrailingWhitespace()` and `endWithNewline()`. Wdyt?

Reply:
addressed, added both suggestions, thanks!